tests/Imagine/Test/Image/AbstractImagineTest.php

```diff
@@ -116,7 +116,7 @@ public function testShouldCreateImageFromString()
 
     public function testShouldCreateImageFromStreamWithMetadata()
     {
-        $source = 'http://imagine.readthedocs.org/en/latest/_static/exit-90-test.jpg';
+        $source = 'https://imagine.readthedocs.org/en/latest/_static/exit-90-test.jpg';
```

Comment:
Most likely you want to test it in testShouldOpenAnHttpImage() and change HTTP_IMAGE to https. Here image is opened by `fopen`.

Reply:
I'll revert this edit. About changing the HTTP_IMAGE constant: it should work even if we leave it with the HTTP protocol (the standard CURL adapter is configured to follow redirects).